simtools/applications/simulate_showers_for_trigger_rates.py

```diff
@@ -141,47 +180,48 @@ def main():
             "data_directory": Path(args_dict["data_directory"]) / label,
             "site": args_dict["site"],
             "layout_name": args_dict["array_layout_name"],
-            "run_range": [1, args_dict["nruns"] + 1],
+            "run_range": [
+                args_dict["run_number"],
+                args_dict["run_number"] + args_dict["nruns"] - 1,
+            ],
             "nshow": args_dict["nevents"],
             "primary": args_dict["primary"],
             "zenith": args_dict["zenith"] * u.deg,
             "azimuth": args_dict["azimuth"] * u.deg,
         },
         "showers": {
-            "erange": [10 * u.GeV, 300 * u.TeV],
-            "eslope": -2,
-            "viewcone": 10 * u.deg,
-            "cscat": [20, 1500 * u.m, 0],
+            "erange": [args_dict["energy_min"] * u.TeV, args_dict["energy_max"] * u.TeV],
+            "eslope": args_dict["e_slope"],
+            "viewcone": args_dict["view_cone"] * u.deg,
+            "cscat": [
+                args_dict["num_use"],
+                args_dict["scatter_x"] * u.m,
+                args_dict["scatter_y"] * u.m,
+            ],
         },
     }
 
     shower_simulator = Simulator(
         label=label,
-        simulation_software="corsika",
+        simulation_software="corsika_simtel",
         simulator_source_path=args_dict.get("simtel_path", None),
         config_data=shower_config_data,
         submit_command=args_dict.get("submit_command", ""),
-        test=args_dict["test"],
         mongo_db_config=db_config,
         model_version=args_dict.get("model_version", None),
     )
 
-    if not args_dict["test"]:
-        shower_simulator.simulate()
-    else:
-        logger.info("Test flag is on - it will not submit any job.")
-        logger.info("This is an example of the run script:")
-        shower_simulator.simulate()
+    if args_dict["array_layout_name"] in ["1MST", "1LST", "1SST"]:
```

Comment:
Would that be the file to use for any single-telescope trigger run? I would suggest making this independent of the telescope type and either a) count the number of telescopes available for a given layout or the add a dedicated command line flag for single telescope trigger runs.

Reply:
Yes, same file for all single telescope arrays. This is a good idea, I implemented it now.